functions/helpers/customChannels/customChannelToFlex.private.ts

```diff
@@ -16,7 +16,14 @@
 
 import { Context } from '@twilio-labs/serverless-runtime-types/types';
 
+export type ConversationSid = `CH${string}`;
+export type ChatChannelSid = `CH${string}`;
+
+const CONVERSATION_CLOSE_TIMEOUT = 'P3D';
```

Comment:
Can you add a comment on what `P3D` means?

Reply:
I will link to the ISO standard wiki page, good call